src/hoi4_world/countries/hoi4_country_converter.cpp

```diff
@@ -473,12 +501,22 @@ std::optional<hoi4::Country> hoi4::ConvertCountry(const vic3::World& source_worl
    {
       overlord = country_mapper.GetHoiTag(*source_overlord);
    }
-
+   Log(LogLevel::Debug) << fmt::format("{}", tag.value_or(""));
    const auto ideology_support = DetermineIdeologySupport(source_country.GetInterestGroupIds(),
        source_world.GetInterestGroups(),
        source_world.GetIdeologies(),
        ideology_mapper);
 
+   if (IDEOLOGY_DEBUG)
+   {
+      Log(LogLevel::Debug) << fmt::format("{}: {} {} {} {}",
+          tag.value_or(""),
+          ideology_support.at("democratic"),
+          ideology_support.at("communism"),
+          ideology_support.at("fascism"),
+          ideology_support.at("neutrality"));
+   }
```

Comment:
Maybe make this an always-on info message? Make it more clear at a glance in the logs though.

Reply:
I don't see this being useful for the user, and I'm not seeing how this would be useful in telemetry for us.
What might be useful is a condensed blob of data that contains like, weights, faction clout, etc, so we could tune better. But that's beyond the scope of this PR.